runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -513,3 +535,40 @@ SOLINK_EXTRA_ARGS+=-Wl,--version-script=$(SOLINK_VERSION_SCRIPT)
 endif
 
 SOLINK_FLAGS+=$(SOLINK_FLAGS_EXTRA)
+
+ifneq ($(JITSERVER_SUPPORT),)
+    #
+    # Setup protobuf
+    #
+    PROTO_CMD?=protoc
+
+    SOLINK_SLINK_STATIC=-l:libprotobuf.a
```

Comment:
Why not just `-lprotobuf`? (I didn't know `-l:xxx` was special.)

Reply:
If using `-lprotobuf`, we'll need to add `-static` to ` SOLINK_SLINK_STATIC`.